ocsp/responder.go

```diff
@@ -85,25 +85,44 @@ type Responder struct {
 // from an OCSP request to an OCSP response is done by the Source;
 // the Responder simply decodes the request, and passes back whatever
 // response is provided by the source.
+// Note: The caller must use http.StripPrefix to strip any path components
```

Comment:
just read the source code of nginx. OK we have to live in an imperfect world. Thanks for the patch.